tests/src/Simple/PreInitData/PreInitData.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
I can't see this in the GitHub UI: does this file have the executable special mode flags (chmod)? Since it's not actually running, not setting them won't be a CI failure.